visualize.Rmd

```diff
@@ -1,115 +1,113 @@
-# Data visualisation
+# Visualización de datos
 
-## Introduction
+## Introducción
 
-> "The simple graph has brought more information to the data analyst’s mind 
-> than any other device." --- John Tukey
+>"Un simple gráfico brinda más información a la mente del analista de datos
+>que cualquier otro dispositivo". --- John Tukey
 
-This chapter will teach you how to visualise your data using ggplot2. R has several systems for making graphs, but ggplot2 is one of the most elegant and most versatile. ggplot2 implements the __grammar of graphics__, a coherent system for describing and building graphs. With ggplot2, you can do more faster by learning one system and applying it in many places.
+En este capítulo aprenderás cómo visualizar datos usando el paquete **ggplot2**. De los muchos sistemas que posee R para hacer gráficos, **ggplot2** es uno de los más elegantes y versátiles. Esto se debe a que implementa  un sistema coherente para describir y construir gráficos, conocido como la __gramática de gráficos__. Con **ggplot2** puedes hacer más cosas en menor tiempo, aprendiendo un único sistema y aplicándolo en muchos lugares
 
-If you'd like to learn more about the theoretical underpinnings of ggplot2 before you start, I'd recommend reading "The Layered Grammar of Graphics", <http://vita.had.co.nz/papers/layered-grammar.pdf>.
+Si deseas obtener más información sobre los fundamentos teóricos de **ggplot2** antes de comenzar, te recomendamos leer "La gramática de gráficos en capas", <http://vita.had.co.nz/papers/layered-grammar.pdf>.
 
-### Prerequisites
+### Prerrequisitos
 
-This chapter focusses on ggplot2, one of the core members of the tidyverse. To access the datasets, help pages, and functions that we will use in this chapter, load the tidyverse by running this code:
+Este capítulo se centra en **ggplot2**, uno de los paquetes  principales de tidyverse. Para acceder a los conjuntos de datos, las páginas de ayuda y las funciones que utilizaremos en este capítulo, debes cargar tidyverse ejecutando este código:
 
 ```{r setup}
 library(tidyverse)
 ```
 
-That one line of code loads the core tidyverse; packages which you will use in almost every data analysis. It also tells you which functions from the tidyverse conflict with functions in base R (or from other packages you might have loaded). 
+Esa única línea de código carga el núcleo de tidyverse; compuesto por los paquetes que usarás en casi todos tus análisis de datos. Al correr esta línea verás cuáles  funciones de tidyverse pueden tener conflicto con funciones de R base (o de otros paquetes que puedas haber cargado previamente). 
```

Comment:
* En el segundo enunciado, *It also tell you* puede utilizarse *Al correr esta línea también verás*.

Reply:
De acuerdo con esta revisión. Propongo también de mi parte unas pequeñas modificaciones:
- *de tidyverse* -> *del tidyverse*
- eliminar doble espacio en *cuáles funciones*

El párrafo quedaría entonces:
 > Esa única línea de código carga el núcleo del tidyverse; compuesto por los paquetes que usarás en casi todos tus análisis de datos. Al correr esta línea también verás cuáles funciones de tidyverse pueden tener conflicto con funciones de R base (o de otros paquetes que puedas haber cargado previamente).